### ISSUE TITLE
Playlist files not ignored when using -d or -i

### ISSUE THREAD

**Issue by mmalisz** [nims11/IPod-Shuffle-4g#37]
Example:
I've got a dir with a m3u file within an directory. When using -d switch, said directory produces 2 playlists: one from the dir, another from the playlist file.
Same for id3 option.

Expected behaviour:
m3u / pls / any other supported playlist type should be ignored when using -d / -i switch.

**Comment by NicoHood** [comment]
You are right. I also have this problem, with the exception that my playlist files are broken and just give errors ;)
We should probably add a new switch for local playlists. something like `--local-playlists` for m3u pls.

**Comment by mmalisz** [comment]
Well, it also makes sense to use local playlists if neither of auto-generating param is passed.

**Comment by NicoHood** [comment]
That is true. Otherwise you'd have nothing. So I'd describe the param as:
`-l, --local-playlists` - `Use m3u and pls playlists even if any of the auto-playlist features are used`

**Comment by mmalisz** [comment]
I meant use local playlists by default if no param has been passed.

**Comment by NicoHood** [comment]
Yes. And this param will explicitly enable them even if autoplaylists are used (where we will disable it then)
